environment_mac.yml

```diff
@@ -32,12 +29,16 @@ dependencies:
   - pillow>=8.0.1
   - scikit-learn
   # 3rd party algorithms for different modules
```

Comment:
Thanks for the PR, @akshay-krishnan.

CUDA is not compatible with Mac so we should leave this line as it was before.

Reply:
Marking as resolved, thanks